app/views/modals/edit-resource.html

```diff
@@ -0,0 +1,101 @@
+<div class="modal-resource-action">
+  <div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-hidden="true" aria-label="Close" ng-click="cancel()">
+      <span class="pficon pficon-close"></span>
+    </button>
+    <h1 class="modal-title">Expand Persistent Volume Claim</h1>
+  </div>
+
+  <div class="modal-body">
+    <h3>Increase the Capacity of claim
+      <strong>{{displayName ? displayName : resourceName}}</strong>.
+      <a href="{{'persistent_volumes' | helpLink}}" target="_blank">
+        <span class="learn-more-inline">Learn More
+          <i class="fa fa-external-link" aria-hidden="true"></i>
+        </span>
+      </a>
+    </h3>
+
+    <p><span>This can be a time-consuming process.</span></p>
+
+    <ng-form name="expandPersistentVolumeClaimForm">
```

Comment:
Why `ng-form` instead of `<form>`?

Reply:
Prob should clarify why we would suggest not using `ng-form` followed by a suggestion to use`ng-href`?